docker/docker-compose.yaml

```diff
@@ -39,7 +39,44 @@ services:
     container_name: unstract-execution-consumer
     restart: unless-stopped
     entrypoint: .venv/bin/celery
-    command: "-A backend worker --loglevel=info -Q celery,celery_periodic_logs,celery_log_task_queue --autoscale=8,1"
+    command: "-A backend worker --loglevel=info -Q celery --autoscale=${EXECUTION_CONSUMER_AUTOSCALE}"
+    env_file:
+      - ../backend/.env
+    depends_on:
+      - redis
+    environment:
+      - ENVIRONMENT=development
+    labels:
+      - traefik.enable=false
+    volumes:
+      - ./workflow_data:/data
+      - ${TOOL_REGISTRY_CONFIG_SRC_PATH}:/data/tool_registry_config
+
+
+  # Celery worker for managing logs and periodic tasks
+  log-management-worker:
+    image: unstract/backend:${VERSION}
+    container_name: unstract-log-management-worker
```

Comment:
@muhammad-ali-e Can we call this `worker-logs` instead?

Reply:
`worker-logs` sounds like logs of worker, right?. I here meant the celery worker for manging logs related process.

Reply:
Renamed to `worker-logging`.